src/components/manage/Widgets/AlignWidget.jsx

```diff
@@ -63,7 +63,10 @@ const AlignWidget = (props) => {
     }
   });
 
-  const icons = { ...ICON_MAP, ...actions_icon_map };
+  const actions_info = {
+    ...default_actions_info({ intl }),
+    ...actions_icon_map,
```

Comment:
@sneridagh since the mapping is no longer only about the icon map but also about the title of the
action, how about naming it **actions_options**? 
It makes more sense to have such a naming as there I can configure the id, the icon to use and the title of the action

Reply:
Fixed.